packages/ui/components/footer/Footer.tsx

```diff
@@ -13,24 +13,42 @@ export interface FooterProps {
 
 export const Footer = ({ links, vercelpoweredProps }: FooterProps) => {
   return (
-      <VStack bg='black' py={{ base:'10px', md:'5px' }} px={{ base:'5px', lg:'70px' }}>
-
+      <VStack
+          bg='black'
+          py={{ base:'50px', md:'5px' }}
+          px={{ base:'5px', md:'70px' }}
+      >
         {/* Main Footer */}
-        <Grid w='100%' py='10px' templateColumns={ { base: 'repeat(1, 1fr)', md: 'repeat(14, 1fr)' } } gap={ 4 }>
+        <Flex
+            w='100%'
+            py='10px'
+            flexDir={{ base:"column", md:"row" }}
+            justifyContent="space-between"
+            alignItems="center"
+        >
           {/* Vercel Tag */}
-          <GridItem colSpan={ { base:1, md: 5 } }>
+          <Box pt={ { base:"10px" } } pb={ { base:"30px" } }>
             <Flex justifyContent={{ base:'center', md:'flex-start' }} alignItems='center'>
               <VercelPowered {...vercelpoweredProps} />
```

Comment:
is it possible to align this vercel tag to the middle vertically? (I think is because of the pb='30px')
![image](https://user-images.githubusercontent.com/47933193/201489072-4a353684-6ad0-4321-bbda-8647792d7687.png)